### DISCUSSION TITLE
No save icon in site preview list

### DISCUSSION THREAD

**Discussion by vinespie** [akeeba/panopticon#765] [Q&A (Community Support)] [answered]
How can the display of the icon indicating no save be explained when a save is clearly visible in the site details?
I recreated the site and reloaded the save information, but it doesn't change anything.

![image](https://github.com/user-attachments/assets/6ccaaff3-223c-43e4-a720-ad3ffd4638b9)

![image](https://github.com/user-attachments/assets/0b2bb492-efff-4128-9435-82482d3ccb3e)

**Comment by nikosdion** [comment]
Did you try reloading the list of backups for the site?

**Reply by vinespie** [reply]
I clicked on these two buttons, but I'm not sure if that reloads the list of saves or just the list of profiles.
![image](https://github.com/user-attachments/assets/f55a0c68-3c20-4708-a3a4-02ddcb58fbfa)

**Reply by vinespie** [reply]
@nikosdion I looked in the documentation and did not find any information.

**Reply by nikosdion** [reply]
If you click the second button it reloads the list of backups.

I have a live installation of Panopticon monitoring real sites which also use Akeeba Backup Professional to save backups to remote storage. It correctly shows the last backup status as cloud storage, not never taken. Are you sure you are looking at the right site?

**Reply by vinespie** [reply]
Yes, this is the right site. To fix it, I have scheduled a backup in Panopticon. When the backup is complete, the icon is correct.

**Reply by nikosdion** [reply]
The icon you see in the Sites Overview page is information cached whenever Panopticon refreshes the site's information (default is once every 15 minutes).

**Reply by vinespie** [reply]
I have about 900 sites in Panopticon, each with scheduled backups. Can this 15-minute delay be exceeded if there are many tasks for Panopticon to execute? Is there a notion of priority for tasks?

**Reply by nikosdion** [reply, marked as answer]
Please read https://github.com/akeeba/panopticon/wiki/CRON-Jobs

With so many monitored sites you should have at least 30 CRON jobs and stagger your backups. Otherwise you end up in a situation where all available task execution time is consumed by taking backups and everything else starts sliding back.
